### PR TITLE
Added support for php 5.3- and old methods

### PR DESCRIPTION
**Pull request by pur4u**
There is no task for that.
I have added the old methods and have made the php support less than 5.3.

### CODE REVIEW

File: classes/cf-input.class.php [hunk 1]
```diff
@@ -271,6 +271,17 @@ function __construct( $conf ) {
 			$this->value = $this->get_value();
 			return true;
 		}
+
+		/**
+		 * Added support for php 5.3-
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Comment should end with a `.`.

File: classes/cf-input.class.php [hunk 2]
```diff
@@ -271,6 +271,17 @@ function __construct( $conf ) {
...
+		 *
+		 * @param $conf
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Missing parameter type and documentation.

File: classes/cf-input-settings-types.class.php [hunk 3]
```diff
@@ -9,7 +9,18 @@
...
+	}
+
+	/**
+	 * Added support for php 5.3-
+	 *
+	 * @param $conf
+	 *
+	 * @return cf_input_block
+	 */
+	function cfs_input_block( $conf ) {
```
Comment (mbucurcf) [inline classes/cf-input-settings-types.class.php]: Missing function visibility. I know it's `public` by default, we should make that explicit.

File: classes/cf-input.class.php [hunk 4]
```diff
@@ -271,6 +271,17 @@ function __construct( $conf ) {
...
+		 *
+		 * @return mixed
+		 */
+		function cf_input( $conf ) {
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Missing function visibility. I know it's `public` by default, we should make that explicit.

File: classes/cf-input.class.php [hunk 5]
```diff
@@ -480,6 +491,17 @@ class cf_input_text extends cf_input {
 		function __construct( $conf ) {
 			return cf_input::cf_input($conf);
 		}
+
+		/**
+		 * Added support for php 5.3-
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Comment should end with a dot.

File: classes/cf-input.class.php [hunk 6]
```diff
@@ -480,6 +491,17 @@ class cf_input_text extends cf_input {
...
+		 *
+		 * @param $conf
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Missing parameter type and documentation.

File: classes/cf-input-settings-types.class.php [hunk 7]
```diff
@@ -9,7 +9,18 @@
 */
 class cfs_input_block extends cf_input_block {
 	function __construct( $conf ) {
-		return cf_input_block::cf_input_block($conf);
+		return cf_input_block::cf_input_block( $conf );
+	}
+
+	/**
+	 * Added support for php 5.3-
```
Comment (mbucurcf) [inline classes/cf-input-settings-types.class.php]: You should comment what the function does, not that you added support for PHP 5.3.

**Review by bucur-marian — CHANGES_REQUESTED** [review]
Changes requested.

File: classes/cf-input.class.php [hunk 8]
```diff
@@ -271,6 +271,17 @@ function __construct( $conf ) {
 			$this->value = $this->get_value();
 			return true;
 		}
+
+		/**
+		 * Added support for php 5.3-.
+		 *
+		 * @param array $conf  Config array for this element.
```
Comment (mbucurcf) [inline classes/cf-input.class.php]: Please fix extra spacing between $conf and Config. Same for below.

**Pull request closed without merging**